### PR TITLE
Make builtin the default migrations backend

### PR DESCRIPTION
**Pull request by markstory**
The 4.3 branch has been out for a few months and there haven't been any showstopping bugs reported yet. Changing the default backend to builtin and retaining the phinx backend as an escape hatch until a fix can be released seems like a decent compromise on moving forwards and retaining compatibility.

### CODE REVIEW

File: src/Command/MigrateCommand.php [hunk 1]
```diff
@@ -80,6 +80,7 @@ public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionPar
             'help' => "Mark any migrations selected as run, but don't actually execute them",
             'boolean' => true,
         ])->addOption('dry-run', [
+            'short' => 'x',
```
Comment (othercorey) [inline src/Command/MigrateCommand.php:83]: Was this change intentional?

Comment (markstory) [reply src/Command/MigrateCommand.php:83]: Yes. I missed this short alias and the tests for CompletionShell integration identified the regression.
